src/user.js

```diff
@@ -78,7 +78,7 @@ async function markUserStaleBeforeNow(username) {
 }
 
 
-module.exports = {
+export {
   userStaleBefore,
   markUserStaleBeforeNow,
 };
```

Comment:
I'd also be happy to refactor these end-of-file exports to be "inline" exports, but that can be opinionated. Won't do it unless I get a thumbs up from someone else here.

Reply:
I looked for an ESLint rule to prefer inline exports over end-of-file exports, but could not find one. I only found [import/exports-last](https://github.com/import-js/eslint-plugin-import/blob/06cc47c4e60246a4caa120b2d8cc0a98e07e894d/docs/rules/exports-last.md) which prefers end-of-file over inline.

Now I'm thinking that maybe it's better to keep as-is and enforce `import/exports-last` for future consistency.

Reply:
Hmm. I wouldn't take ESLint as gospel here, although I know that opinion itself is blasphemy in some circles. I don't care too much about the export styles we use (maybe I don't know enough about vagaries of either to care), but I'd have a slight preference to not enforce one or the other. For example, it can be nice to use inline exports as you can see what's exported and what's not when reading the file without jumping to the bottom.

Reply:
I personally find smaller files with inline exports easier to understand. But I don't have a strong preference or will to make a large refactor here, so I'll keep things as-is without adding an `import/exports-last` rule.